### PR TITLE
Update CSV with profiles and remove extra parameters

### PR DESCRIPTION
**Pull request by damemi**

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/cc @soltysh @ingvagabund 
minor update for the operatorhub description

File: manifests/4.7/cluster-kube-descheduler-operator.v4.7.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -49,29 +49,23 @@ spec:
...
+      * AffinityAndTaints
+      * TopologyAndDuplicates
+      * LifecycleAndUtilization
 
+      These profiles are documented in detail in the [descheduler README](https://github.com/kubernetes-sigs/descheduler/#profiles).
+      
       ## Additional Parameters
-
+      
       In addition to the strategies, the following parameters can be configured:
```
Comment (ingvagabund) [inline manifests/4.7/cluster-kube-descheduler-operator.v4.7.0.clusterserviceversion.yaml:66]: `In addition to the strategies` -> `In addition to the profiles`

Comment (damemi) [reply manifests/4.7/cluster-kube-descheduler-operator.v4.7.0.clusterserviceversion.yaml]: @ingvagabund thanks, fixed.

File: manifests/4.7/cluster-kube-descheduler-operator.v4.7.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -49,29 +49,23 @@ spec:
...
-      * `PodLifeTime` - This strategy evicts pods that are older than a specified time
-
-      These strategies are documented in detail in the [descheduler README](https://github.com/kubernetes-sigs/descheduler/#policy-and-strategies).
+      The following profiles are available:
+      
+      * AffinityAndTaints
+      * TopologyAndDuplicates
+      * LifecycleAndUtilization
 
+      These profiles are documented in detail in the [descheduler README](https://github.com/openshift/cluster-kube-descheduler-operator#profiles).
```
Comment (ingvagabund) [inline manifests/4.7/cluster-kube-descheduler-operator.v4.7.0.clusterserviceversion.yaml:62]: s/descheduler/descheduler operator

**Review by ingvagabund — CHANGES_REQUESTED** [review]

File: manifests/4.7/cluster-kube-descheduler-operator.v4.7.0.clusterserviceversion.yaml (hunk 2 above)
Comment (damemi) [reply manifests/4.7/cluster-kube-descheduler-operator.v4.7.0.clusterserviceversion.yaml]: done

**Pull request merged**